backend/api/Areas/Admin/Mapping/User/UserMap.cs

```diff
@@ -40,7 +38,15 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.Note, src => src.Note)
                 .Map(dest => dest.Agencies, src => src.Agencies)
                 .Map(dest => dest.Roles, src => src.Roles)
+                .AfterMappingInline((m, e) => UpdateUser(m, e))
```

Comment:
I haven't seen this before, what is this doing?

Reply:
It allows me to set the UserId correctly.  Essentially without it the child objects in a collection don't have access to the parent object.